server/db/connection.js

```diff
@@ -18,6 +20,25 @@ try {
             //         db.Test.find().then((data) => console.log(data));
             //     }
             // );
+
+            // testing appointment model
```

Comment:
I think we can remove these commented out tests, unless you want to keep them for specific testing in the future and then remove them